k8s/upgrades/0.9.0-1.0.0/jiva/jiva_volume_upgrade_auto.sh

```diff
@@ -0,0 +1,250 @@
+#!/bin/bash
+
+################################################################
+# STEP: Get Persistent Volume (PV) name as argument            #
+#                                                              #
+# NOTES: Obtain the pv to upgrade via "kubectl get pv"         #
+################################################################
+
+function on_exit() {
+    echo "Clearing temporary files"
+    rm jiva-replica-patch.json
```

Comment:
if files are not present then what?
why dont we create a folder like `upgrade_tmp` that contains all these files

Reply:
Done.